content/en/blog/_posts/2024-10-01-migrate-cgroup-v2.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: blog
+title: 'The Shift to cgroup v2 in Kubernetes: What You Need to Know'
+date: 2024-10-01
+slug: kubernetes-cgroup-v2-shift
```

Comment:
it should be `cgroups` plural, not a singular `cgroup`
https://en.wikipedia.org/wiki/Cgroups

applies to a number of places in the doc.

Reply:
Does this mean that all cgroup should be cgroups?

Reply:
It should be `cgroup` according to kernel docs https://docs.kernel.org/admin-guide/cgroup-v2.html